botx/testing/building/builder.py

```diff
@@ -128,3 +141,143 @@ def mention_chat(self, chat_id: uuid.UUID) -> None:
                 ),
             ),
         )
+
+    def image(
+        self,
+        content: str = "data:text/plain;base64,",  # noqa: WPS110
```

Comment:
move `"data:text/plain;base64,"` under separate module constant. and maybe use real base64 content here, so default might be really useful